File: beginner/python/1159.py
while True:
    soma = 0
    cont = 0
    x = int(input())

    if (x == 0):
        break

    while cont < 5:
        if (x % 2 == 0):
            soma += x
            cont+=1
    
        x+=1

    print(soma)

File: beginner/javascript/1159.js
var input = require('fs').readFileSync('stdin', 'utf8');
var linhas = input.split('\n');

var x, soma, cont;

while (true) {
    soma = 0;
    cont = 0;
    x = parseInt(linhas.shift());

    if (x == 0)
        break;

    while (cont < 5) {
        if (x % 2 == 0) {
            soma += x;
            cont++;
        }
        x++;
    }

    console.log(soma);
}